### PR TITLE
{vis}[intel-2016a] FLTK 1.3.3 with libX11 1.6.3 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# author: Dina Mahmoud Ibrahim ( Cairo University )
+#
+easyblock = 'ConfigureMake'
+
+name = 'FLTK'
+version = '1.3.3'
+
+libx11 = 'libX11'
+libxver = '1.6.3'
+versionsuffix = '-%s-%s' % (libx11, libxver)
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb]: I'm not sure if this is necessary?

It's OK to include `libX11` as a dep and not use a `versionsuffix` imho...

File: easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    ('Tk', '8.6.4', versionsuffix),
+    ('Mesa', '11.1.2'),
+    ('libGLU', '9.0.0'),
+    ('libpng', '1.6.21'),
+    ('libjpeg-turbo', '1.4.2'),
+    ('xprop', '1.2.2'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/fltk-config','bin/fluid'],
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb]: space after `,`

File: easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+sources = ['%(namelower)s-%(version)s-source.tar.gz']
+source_urls = ['http://fltk.org/pub/%(namelower)s/%(version)s/']
+
+
+dependencies = [
+    (libx11, libxver),
+    ('libXcursor', '1.1.14'),
+    ('libXinerama', '1.1.3'),
+    ('Tcl', '8.6.4'),
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb]: are you sure bumping from `8.5.x` to `8.6.x` is OK? It's not backwards compatible.

File: easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb (hunk 1 above)
Comment (hajgato) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb]: @boegel dunno. It is a drastic change, but I can leave it out.

Comment (boegel) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb]: I would leave it out...

File: easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb (hunk 3 above)
Comment (hajgato) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb]: @boegel: no idea. I am not even sure that `Tcl/Tk` is needed. 
I think `Tcl/TK` have an `FLTK` extension (http://wiki.tcl.tk/1826, and http://sourceforge.net/projects/tclfltk), but `FLTK` does not even needs `Tcl/Tk`

Comment (boegel) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb]: well, shouldn't we remove them then?

Comment (hajgato) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.3-intel-2016a-libX11-1.6.3.eb]: I guess, yes.

**Comment by hajgato** [comment]
@boegel: Tcl/Tk dep removed

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f3962d4dc01fdc6c688c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2090.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e5d238951684d1020f64 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**